.github/workflows/atlantis-image.yml

```diff
@@ -12,6 +12,20 @@ on:
       - 'Dockerfile'
       - '.github/workflows/atlantis-image.yml'
   workflow_dispatch:
+  # The workflow_run method is the ideal method to use but it's complicated due to
+  # the usage of a matrix in the both the base and image workflows. It's possible
+  # that the base debian may finish before the base alpine and that may trigger
+  # both workflows.
+  #
+  # workflow_run:
+  #   workflows: ["atlantis-base"]
+  #   types: [completed]
+  #   branches: [main]
```

Comment:
Actually I think it is a good idea, it should be put into the base flow and trigger this workflow upon success completion.